packages/components/src/components/details/details.custom.styles.ts

```diff
@@ -0,0 +1,114 @@
+import { css } from 'lit';
+
+export default css`
+  .details {
+    border-color: var(--syn-color-neutral-400);
+    border-radius: var(--syn-border-radius-none);
+    border-width: 0 0 var(--syn-border-width-small);
+  }
+
+  /**
+   * We do not want to have a opaque border-color,
+   * so we apply the opacity to the inner elements instead
+   */
+  .details--disabled {
+    opacity: 1;
+  }
+
+  .details--disabled summary,
+  .details--disabled .details__body {
+    opacity: var(--syn-opacity-50);
+  }
```

Comment:
@LeaLeRose I'm not sure if this is good. IMO we should stick to our current convention and make the whole element look disabled, including the border at the bottom.